sqllogictest/src/runner.rs

```diff
@@ -391,8 +391,8 @@ fn format_diff(
 ///
 /// # Default
 ///
-/// By default, we will use `|x, y| x == y`.
-pub type Validator = fn(&Vec<String>, &Vec<String>) -> bool;
+/// By default, we will use compare normalized results.
+pub type Validator = fn(&Vec<Vec<String>>, &Vec<String>) -> bool;
```

Comment:
This signature change LGTM, although I think it's not very necessary. 🤣
If you can split expected results, you can indeed split actual results?

Reply:
Oh I'm wrong, actual results are normalized before passed to `Validator`, which makes splitting not possible.